app/views/contact_information/edit.html.erb

```diff
@@ -8,7 +8,7 @@
   </header>
   <div class="form-card__content">
     <%= form_for @step, as: :step, builder: MbFormBuilder, url: current_path, method: :put do |f| %>
-      <%= f.mb_input_field :phone_number, "What is the best phone number to reach you?", options:  { placeholder: "Phone #" } %>
+      <%= f.mb_input_field :phone_number, "What is the best phone number to reach you?" %>
```

Comment:
Copy: "What is the best phone number _at which_ to reach you?" maybe?

Reply:
I think that's a good suggestion but I will let this one go for now since we will do a copy audit soon anyway. I know simplicity of language is a big goal.